osf-cli/src/main/java/org/dataconservancy/cos/packaging/cli/PackageGenerationApp.java

```diff
@@ -16,13 +16,26 @@
 
 package org.dataconservancy.cos.packaging.cli;
 
-import java.io.File;
-import java.net.URL;
-
+import java.io.*;
```

Comment:
Same here, can we remove wildcard imports?  There is an IDEA setting to help prevent this.